### PR TITLE
ref: Move all symbolicator-related code in store.py to a separate file (NATIVE-259)

### PR DESCRIPTION
**Pull request by relaxolotl**
aka reduce the number of angry markus "why is my review requested on this" comments on native PRs.

Ingest can stop being pulled in for reviews on PRs that do not
require their feedback or should not require their feedback.

### CODE REVIEW

File: src/sentry/tasks/store.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import logging
```
Comment (untitaker) [inline src/sentry/tasks/store.py:1]: From this file you will have to temporarily re-export all symbolication tasks as there can be in-flight queue items for which the worker will attempt to import from this module.

```suggestion
import logging
from sentry.tasks.symbolication import symbolicate_event, ...
```

**Review by untitaker — COMMENTED** [review]
thank you!

**Comment by flub** [comment]
generally LGTM.  I put this in NATIVE-259 if you want to link the ticket btw

**Review by loewenheim — APPROVED** [review]

File: src/sentry/tasks/symbolication.py [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    project_id = data["project"]
+    set_current_event_project(project_id)
+
+    event_id = data["event_id"]
+
+    from_reprocessing = symbolicate_task is symbolicate_event_from_reprocessing
+
+    def _continue_to_process_event():
+        process_task = process_event_from_reprocessing if from_reprocessing else process_event
+        # TODO: this uses a private "store" function. is there a way to not do this?
```
Comment (relaxolotl) [inline src/sentry/tasks/symbolication.py:91]: nts: figure out an answer to this, namely "is this ok?" and how to make it ok if it isn't

also probably good to think about the the line that has been drawn by splitting all of the symbolicator stuff out into its own file: are there other places where it's using internal `store.py` stuff? is there symbolication logic in `store.py` that could and should be moved out into a method in this file? what are the low hanging fruit for those?

File: src/sentry/tasks/store.py (hunk 1 above)
Comment (untitaker) [reply src/sentry/tasks/store.py:1]: actually add yourself to https://github.com/getsentry/sentry/blob/a2212cd3f2cae4411c7c06bcbcecf03ba7bac2d7/src/sentry/conf/server.py#L543 instead

**Review by flub — COMMENTED** [review]
LGTM, did we test locally if events still get processed and symbolicated etc?

**Comment by relaxolotl** [comment]
@untitaker can you take a second look at this when you have time?

**Review by untitaker — CHANGES_REQUESTED** [review]
there is one change to the task `name` attribute that really needs to be reverted and tested (probably manually), otherwise I believe this is good to go

**Review by flub — CHANGES_REQUESTED** [review]
https://github.com/getsentry/sentry/blob/b576d6fcd52fb8395e329a3f27b16a60ff4bddac/src/sentry/api/serializers/models/project.py#L44 also needs to be updated with this move

**Review by untitaker — APPROVED** [review]

**Review by flub — APPROVED** [review]

**Pull request merged**